src/Databases/DatabaseAtomic.cpp

```diff
@@ -269,6 +269,7 @@ void DatabaseAtomic::commitCreateTable(const ASTCreateQuery & query, const Stora
                             database_name, query.database);
         not_in_use = cleanupDetachedTables();
         assertDetachedTableNotInUse(query.uuid);
+        DatabaseCatalog::instance().addUUIDMapping(query.uuid, {}, {});
```

Comment:
Not clear, why do we need it? We will "fill" this UUID in `attachTableUnlocked`.